README.md

```diff
@@ -1 +1,114 @@
-# my_all
+# Code Along: Building the `all?` method 
+
+##Objectives
+* Continue to practice using yield and blocks
+* Gain a deeper understanding of `#all?`
+
+
+##Code Along Exercise
+
+Fork and clone this lab. You'll be coding your solution in `lib/my_all.rb`. If at any point you want to follow along with the example, you can put a `binding.pry` anywhere in the method and run `ruby bin/my_all` your terminal to pry around.
+
+###Step 1
+We need to iterate through each element in the passed in collection using the simple lower level iterator `while`. Remember, since this is a lower level iterator, we will need to explicitly add a counter we'll call `i`, and increment it each time we go into the loop (`i=i+1`):
+
+```ruby
+def my_all?(collection)
+  i = 0
+  while i < collection.length
+    i = i + 1
+  end
+end
+```
+
+
+###Step 2
+For each element in the collection, you're going to `yield` to the block. This is a tough concept to grasp. Let's look at the code and break down what's happening.
```

Comment:
`For each element in the collection, you're going to `yield` to the block` is awkward. `We're going to `yield` each element in the collection to the block`. I'd also leave out the toughness.